specification.rst

```diff
@@ -2652,7 +2675,10 @@ review-book
     reviewed book
 
 song
-    An audio recording. 
+    An audio recording;
+    Can be used for any audio recording (not only music);
```

Comment:
not sure about this. need it be a recording of a song performance (which is implicit here)?

Reply:
It's implemented in a general way in Zotero (as "Audio Recording"), and `song` seems to fill the same broad role for audio recordings as `motion_picture` does for video recordings. 

Thinking in terms of citation needs, diverse audio recordings generally have the same citation forms. Here, for example, are the example audio recording citation forms from the APA manual:
Bach, J. S. (2010). The Brandenburg concertos: Concertos BWV 1043 & 1060 [Album recorded by Academy of St Martin in the Fields]. Decca. (Original work published 1721)
Beethoven, L. van. (2012). Symphony No. 3 in E-flat major [Song recorded by Staatskapelle Dresden]. In Beethoven: Complete symphonies. Brilliant Classics. (Original work published 1804)
Beyoncé. (2016). Formation [Song]. In Lemonade. Parkwood; Columbia.
Bowie, D. (2016). Blackstar [Album]. Columbia.
Childish Gambino. (2018). This is America [Song]. mcDJ; RCA.
King, M. L., Jr. (1963, August 28). I have a dream [Speech audio recording]. American Rhetoric. https://www.americanrhetoric.com/speeches/mlkihaveadream.htm
Lamar, K. (2017). Humble [Song]. In Damn. Aftermath Entertainment; Interscope Records; Top Dawg Entertainment.

All of those can be handled easily with the existing `song` type. 

If `song` is specifically limited to musical recordings, then we would need to a new `audio` type or similar for poetry recordings, recordings of traffic noises, recordings of speeches. That doesn't seem necessary, and it also doesn't follow existing cataloguing practice--an ISRC number can refer to any type of audio or visual recording. 

I guess in sum, I don't think we have to be too caught up on the precise word used for type labels. `speech` is used more broadly than just speeches, for example, but to a variety of types of presentations. Even within music, a `song` might refer to an album, or a symphony,